doc/rose-rug-advanced-tutorials-suicide.html

```diff
@@ -260,6 +260,285 @@ <h3 class="alwayshidden">Note on suicide triggers (2)</h3>
         <li>triggering off a cleanup task in the suite</li>
       </ul>
     </div>
+
+    <div class="slide">
+          <h2 id="advanced-example">Advanced suicide triggers example</h2>
+      <p>The following example is one suite but with three possible 
+      scenarios:</p>
+
+      <ul>
+        <li>If a memeber of a family fails carry on.
+        If a specified task fails go to the end of that cycle and remove the 
+        failed task from from the task pool.</li>
+        <li>Check the failure and if it meets the criteria then try a recovery 
+        task. 
+        If that task success then the suite carrys on else if it fails then the 
+        suite stops as it cannot be recovered.</li>
+        <li>If the first task fails got the end of the cycle or if two 
+        specified task fail go to the second from last task and then to the end 
+        of that cycle.</li>
+
+    <div class="slide">
+        <h3 id="example-task2">Advanced example</h3>
+
+      <p>Create a <a href="rose-rug-brief-tour#cli">new suite</a> (or just a
+      new directory somewhere - e.g. in your homespace) containing a
+      <samp>rose-suite.conf</samp> file that looks like this:</p>
+    </div>
+
+    <div class="slide">
+        <h3 class="alwayshidden">Advanced example rose-suite.conf</h3>
+<pre class="prettyprint lang-cylc">
+[jinja2:suite.rc]
+CHECK_RECOVER=False
+GO_TO_END=False
+TWO_FAIL=True
+</pre>
+    <p>To run the different examples of the advanced suicide trigger examples
+    change which variable in the <samp>rose-suite.conf</samp> is set to 
+    <var>True</var>.
+    Remember you can only have one set to <var>True</var> and the other two 
+    are to be set to <var>False</var>.
```

Comment:
Feels like this should be a single variable then with three different possible values.